src/gui/gui_msgs.h

```diff
@@ -37,45 +37,62 @@ constexpr char help_msg[] =
 
 List of common options:
 
-  --printconf          Print the location of the default configuration file.
+  --printconf              Print the location of the default configuration file.
 
-  --editconf           Open the default configuration file in a text editor.
+  --editconf               Open the default configuration file in a text editor.
 
-  -c <command>         Run the specified DOS command before running FILE.
-                       Multiple commands can be specified.
+  --eraseconf              Delete the configuration file.
 
-  -noautoexec          Don't perform any [autoexec] actions.
+  --erasemapper            Delete the mapper file.
 
-  -noprimaryconf       Don't read settings from the primary configuration file
-                       located in your user folder.
+  --startmapper            Run the mapper GUI.
 
-  -nolocalconf         Don't read settings from "dosbox.conf" if present in
-                       the current working directory.
+  --securemode             Start in secure mode.
 
-  -conf <configfile>   Start with the options specified in <configfile>.
-                       Multiple configfiles can be specified.
+  --noautoexec             Ignore autoexec section in configuration file.
 
-  --working-dir <path> Set working directory to <path>. DOSBox will act as if
-                       started from this directory.
+  --opencaptures           Launches some external program (I don't even know...)
```

Comment:
This runs the following function (note the TODO comment).  It looks like the user is supposed to specify an external program they want to run against the captures directory.  Not sure what the purpose of this is.

https://github.com/dosbox-staging/dosbox-staging/blob/1930f7f3cd92d2615b5e65059c2ac7350ca85475/src/gui/sdlmain.cpp#L4683-L4706

Reply:
It' a bizarre feature... Let's just say the same thing as the X manual:

> Calls program with as first parameter the location of the captures folder.

Reply:
Sounds good.